src/zoid/buttons/component.jsx

```diff
@@ -811,6 +812,17 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
         type: "function",
         required: false,
       },
+
+      disableSetCookie: {
+        type: "boolean",
+        queryParam: true,
+        required: false,
+        value: () => {
+          if (getDisableSetCookie()) {
+            return true;
+          }
+        },
```

Comment:
could this be refactored as:
```suggestion
        value: () => getDisableSetCookie(),
```

Reply:
From what I remember with zoid props that use `queryParam: true`, a boolean value will always be in the query params (`disableSetCookie=true` and `disableSetCookie=false`). If we return `undefined` it will not be in the query param (`disableSetCookie=true` or nothing). I think either solution works, but if we want to minimize the size of the query string, we should return `undefined` when false.

Reply:
Actually I may be wrong about this behavior. Ignore the above comment. @xuanwan05 is going to test this out locally and we may be able to go with:
```js
disableSetCookie: {
    type: "boolean",
    queryParam: true,
    required: false,
    value: getDisableSetCookie
},
```

Reply:
oh i just assumed `getDisableSetCookie` returned true or false. If that's not the case, we could make sure it does.